src/meta/service/src/stream_service.rs

```diff
@@ -461,4 +461,35 @@ impl StreamManagerService for StreamServiceImpl {
             .await?;
         Ok(Response::new(ListRateLimitsResponse { rate_limits }))
     }
+
+    async fn alter_sink_config(
+        &self,
+        request: Request<AlterSinkConfigRequest>,
+    ) -> Result<Response<AlterSinkConfigResponse>, Status> {
+        let request = request.into_inner();
+
+        let actor_to_apply = self
```

Comment:
please verify the new sink config is valid first.

Reply:
do it in update_sink_config_by_sink_id